src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -291,13 +357,31 @@ public void updateFilteredBlacklistedPersonList(Predicate<ReadOnlyPerson> predic
         filteredBlacklistedPersons.setPredicate(predicate);
     }
 
+    /**
+     * Obtains the latest list of whitelisted persons from masterlist and adds to {@code filteredWhitelistedPersons}
+     * Raises an {@code event} to signal the requirement for change in displayed list in {@code PersonListPanel}
```

Comment:
Does this method raise an event? Or does a method called in this method raise the event?  if it is the latter, this line should not be here

Reply:
Noted.